cmd/validation.go

```diff
@@ -170,6 +183,31 @@ func validateExecutorArg(executor string) (string, error) {
 	return executorType, nil
 }
 
+func validateReleaseName(releaseName string) error {
+	const reg = "^[a-z0-9]([a-z0-9-]+)?[a-z0-9]$"
+	const maxCharacters = 45
+	errs := []string{}
+
+	if len(releaseName) > maxCharacters {
+		errs = append(errs, "must not contain more than 45 characters")
+	}
+
+	re, err := regexp.Compile(reg)
```

Comment:
nitpick: given the regex is a const expression, it might be better to place the compiled regex as a package const or variable, in case it needs to be reused or referred to in multiple places.

`var releaseNameRegex = regexp.MustCompile("^[a-z0-9]([a-z0-9-]+)?[a-z0-9]$")`